### ISSUE TITLE
Move rel/2.0.0-preview1 branches to rel/2.0.0-preview1 tags

### ISSUE THREAD

**Issue by Eilon** [aspnet/Universe#510]
1. [x] Move https://github.com/aspnet/Mvc-Private/ patch branches into https://github.com/aspnet/Mvc/
2. [ ] Move rel/2.0.0-preview1 branches to rel/2.0.0-preview1 tags

**Comment by smitpatel** [comment]
Also leftover patch branches from 1.0.x & 1.1.x releases?

**Comment by Eilon** [comment]
1. [ ] Move https://github.com/aspnet/Mvc-Private/ patch branches into https://github.com/aspnet/Mvc/
2. [ ] Move rel/2.0.0-preview1 branches to rel/2.0.0-preview1 tags

**Comment by Eilon** [comment]
@smitpatel hmm not sure... I would assume so?

**Comment by smitpatel** [comment]
@Eilon - Can someone from MVC team take up migrating the branches? Branch names are ambiguous between both repo to easily locate which branch to merge and also the patch branches are not merged to dev properly

**Comment by Eilon** [comment]
@kichalla - can you work with @smitpatel on this so we can get all the preview1 stuff with the right tags?

Let me know if you guys have questions on any "special cases" (if there are any).

**Comment by smitpatel** [comment]
I believe, getting branches merged properly to dev would be required before we branch for preview2. Tag creation is not that urgent.

**Comment by pranavkm** [comment]
> also the patch branches are not merged to dev properly

We typically don't merge patch branches back in to dev. Why is that causing issues here?

**Comment by smitpatel** [comment]
> We typically don't merge patch branches back in to dev. Why is that causing issues here?

😕 Then why do we have so many merge commits?

**Comment by kichalla** [comment]
I moved the Mvc-Private branches to Mvc

**Comment by smitpatel** [comment]
Finished Repos
- Mvc
- HttpAbstractions
- DependencyInjection (pending tag push)

**Comment by Eilon** [comment]
What else is left on this issue?

**Comment by smitpatel** [comment]
A lot of repos are pending. Last I was doing this and hit sync issues with GitHub.
This does not affect any source code so I was prioritizing my efforts on bug fixes till we reach code freeze.

**Comment by Eilon** [comment]
@smitpatel ah ok thanks, this is definitely lower priority 😄

**Comment by smitpatel** [comment]
**Branches of KoreBuild converted to tags**
rel/1.0.1
rel/1.0.4
rel/1.0.5
rel/1.1.0-preview1
rel/1.1.0
rel/1.1.2
rel/2.0.0-preview1
rel/2.0.0-preview2
(versioning in KoreBuild is a mess)

**Following repo created/updated tags**
All branches with rel/* following our normal semvar converted to tag. Branches which did not follow normal naming convention or were appended with msbuild were not converted.
All branches which were made tag update nuget.config to point to nuget & aspnetcore-master feed.
Branches which have associated tags are deleted.
Antiforgery
AzureIntegration
BasicMiddleware
BrowserLink
Caching
Common
Configuration
DataProtection
DependencyInjection
Diagnostics
DotNetTools
EntityFramework
EntityFramework.Tools
EventNotification
FileSystem
Hosting
HtmlAbstractions
HttpAbstractions
HttpSysServer
Identity
IISIntegration
JavaScriptServices
JsonPatch
KestrelHttpServer
Localization
Logging
MetaPackages
Microsoft.Data.Sqlite
MusicStore
Mvc
MvcPrecompilation
Options
Proxy
Razor
ResponseCaching
Routing
Scaffolding
Security
ServerTests
Session
SignalR
StaticFiles
Testing
WebSockets
Universe

**Following repos had no changes done**
(Not sure if it was needed)
SignalR-Server
SignalR-Client-JS
SignalR-Client-Java
SignalR-Client-Net
Entropy
Home
BugTracker


**Tags may or may not build**
Given amount of time taken to build each branch in every repo and installing pre-req like VS 2015 etc., none of the tag were tested for build-ability. Given that we can always update the tag later and with v2.0 releasing people are hardly likely to build old branches, there is very minimal value in testing out if all the tags build correctly. We can defer that till actual customer hits an issue and fix if any error then.


I have covered all the repo which are part of universe. If any repo is missed the individual repo owners can get the conversion to tag done easily. Closing this issue as major work has been done.

We need to do post-processing of `rel/2.0.0` to create appropriate tags. Creating new issue to track general post-processing item.
